.travis.yml

```
@@ -23,3 +23,4 @@ script:
     else
       ./mvnw verify -B -V -DskipITs;
     fi;
+  - ./mvnw clean javadoc:jar
```

If you do something like this, you'll want to make sure you trigger the specific javadoc configuration.
```suggestion
  - ./mvnw clean javadoc:jar@attach-javadocs
```

---

Done.